### ISSUE TITLE
django.db.utils.OperationalError: duplicate column name: lft

### ISSUE THREAD

**Issue by aholmback** [divio/django-cms#3986]
Following the tutorial http://docs.django-cms.org/en/support-3.0.x/introduction/install_from_scratch.html, The djangocms-installer terminates with the following error (with version 3.0, it works with 2.4):
```bash
django.db.utils.OperationalError: duplicate column name: lft
```
The error occurs when the script attempts creating the table cms_cmsplugin:
```bash
[...]
Dependencies installed
Syncing...
Creating tables ...
Creating table djangocms_text_ckeditor_text
Creating table auth_permission
Creating table auth_group_permissions
Creating table auth_group
Creating table auth_user_groups
Creating table auth_user_user_permissions
Creating table auth_user
Creating table django_content_type
Creating table django_session
Creating table django_admin_log
Creating table django_site
Creating table cms_usersettings
Creating table cms_placeholder
Creating table cms_cmsplugin
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 415, in handle
    return self.handle_noargs(**options)
  File "/home/alex/env/local/lib/python2.7/site-packages/south/management/commands/syncdb.py", line 92, in handle_noargs
    syncdb.Command().execute(**options)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 415, in handle
    return self.handle_noargs(**options)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/core/management/commands/syncdb.py", line 107, in handle_noargs
    cursor.execute(statement)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/db/backends/util.py", line 69, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/db/backends/util.py", line 53, in execute
    return self.cursor.execute(sql, params)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/db/utils.py", line 99, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/db/backends/util.py", line 51, in execute
    return self.cursor.execute(sql)
  File "/home/alex/env/local/lib/python2.7/site-packages/django/db/backends/sqlite3/base.py", line 450, in execute
    return Database.Cursor.execute(self, query)
django.db.utils.OperationalError: duplicate column name: lft
Creating the projectFailure occurred. Do you want to cleanup by removing /home/alex/tutorial-project? [Y/N]
```

**Comment by JarrenceMelie** [comment]
The same issue...

**Comment by yakky** [comment]
This is caused by django-mptt 0.7. A release to fix the mptt version will be issues soon

**Comment by yakky** [comment]
Fixed in nephila/djangocms-installer/#170
